package.json

```diff
@@ -51,42 +51,47 @@
         ],
         "commands": [
             {
-                "command": "extension.regenerateIcons",
+                "command": "vscode-icons.regenerateIcons",
```

Comment:
I changed the naming to get in line with what other extension maker use. It's kinda a naming convention to avoid conflicts.